### PR TITLE
Update icons everywhere possible

### PR DESCRIPTION
**Pull request by AlexandreSi**

### CODE REVIEW

File: newIDE/app/src/BehaviorsEditor/NewBehaviorDialog.js [hunk 1]
```diff
@@ -340,7 +340,7 @@ export default function NewBehaviorDialog({
                 </Line>
                 <Line justifyContent="center" alignItems="center">
                   <FlatButton
-                    leftIcon={<Create />}
+                    leftIcon={<Edit />}
```
Comment (ClementPasteau) [inline newIDE/app/src/BehaviorsEditor/NewBehaviorDialog.js:343]: just to confirm Create/Edit is the right icon?

File: newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationPreview.js [hunk 2]
```diff
@@ -6,14 +6,14 @@ import { Column } from '../../../UI/Grid';
 import { LineStackLayout, ResponsiveLineStackLayout } from '../../../UI/Layout';
 import ImagePreview from '../../../ResourcesList/ResourcePreview/ImagePreview';
 import Replay from '@material-ui/icons/Replay';
-import PlayArrow from '@material-ui/icons/PlayArrow';
-import Pause from '@material-ui/icons/Pause';
 import Timer from '@material-ui/icons/Timer';
```
Comment (ClementPasteau) [inline newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationPreview.js:9]: why not remove those icons too?

File: newIDE/app/src/UI/Carousel.js [hunk 3]
```diff
@@ -116,10 +116,6 @@ const useStylesForGridListItem = makeStyles(theme =>
   })
 );
 
-const useStylesForLeftArrow = makeStyles({
-  arrow: { '& > path': { transform: 'translate(5px, 0px)' } }, // Translate path inside SVG since MUI icon is not centered
```
Comment (ClementPasteau) [inline newIDE/app/src/UI/Carousel.js:120]: 🔥

**Review by ClementPasteau — APPROVED** [review]

File: newIDE/app/src/BehaviorsEditor/NewBehaviorDialog.js (hunk 1 above)
Comment (AlexandreSi) [reply newIDE/app/src/BehaviorsEditor/NewBehaviorDialog.js:343]: Yes, that's (almost) the same
image
image

**Pull request merged**